src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -934,6 +988,26 @@ SPECTRE_ALWAYS_INLINE auto& get_mutable_reference(
   return box->template get_mutable_reference<Tag>();
 }
 
+/// @{
+/*!
+ * \ingroup DataBoxGroup
+ * \brief Check whether the tag \p Consumer depends on the tag \p
+ * Provider in a given \p Box.
+ *
+ * This is equivalent to the question of whether changing \p Provider
+ * (through `db::mutate`, updating one of its dependencies, etc.)
+ * could change the value of `db::get<Consumer>`.  Tags depend on
+ * themselves, and an item and its subitems all depend on one another.
+ */
+template <typename Provider, typename Consumer, typename Box>
+using tag_depends_on =
```

Comment:
Not a change request, but I will probably get this backwards when I use/read it without looking up the documentation, as I was expecting the first template argument to be the Consumer tag that depends upon the Provider.

Reply:
I copied the order from `tt::is_a<Template, Type>`, but `std::is_base_of<Base, Derived>` is the opposite, so yeah, not obvious which is better.

Reply:
The choice on `is_a` is the same as `is_base_of` in my opinion. It is `<generic, specific>`. That is, the generic template in `is_a` plays a similar role to the generic base class in `is_base_of`. That's why the ordering is that way :shrug:

Reply:
The English is backward, though: "X is base of Y" and "Y is a X".

Reply:
That's fair. I have no preference for what you want to do in this PR :) Just wanted to give context as to why I made `is_a` the way it is!

Reply:
If neither of you have a preference, I would prefer to reverse the order of the template parameters

Reply:
That's fine by me :)